### PR TITLE
Create an expiration date for your offer

### PR DESCRIPTION
**Pull request by ejolles**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java [hunk 1]
```diff
@@ -349,6 +393,78 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+
+    public static class DatePickerFragment extends DialogFragment {
```
Comment (SFulpius) [inline app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java]: If it is only used here, this class should be private.

File: app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java [hunk 2]
```diff
@@ -349,6 +393,78 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+
+        private final Calendar c;
```
Comment (SFulpius) [inline app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java]: You could use better variable name like calendar.

Comment (ejolles) [reply app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java]: Ok thanks!

File: app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java (hunk 1 above)
Comment (ejolles) [reply app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java]: This class should be public and static.

Comment (SFulpius) [reply app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java]: ok

File: app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java (hunk 2 above)
Comment (SFulpius) [reply app/src/main/java/ch/epfl/sweng/swenggolf/offer/CreateOfferActivity.java]: Thank you.

**Pull request merged**